infer_structcol/inference.py

```diff
@@ -7,9 +7,17 @@
 import pandas as pd
 import emcee
 import lmfit
-from .model import (calc_model_spect, calc_resid_spect, min_phi, max_phi, 
-                    min_radius, max_radius, min_thickness, max_thickness, 
-                    min_l0, max_l0, min_l1, max_l1, log_posterior)
+from .model import (calc_model_spect, calc_resid_spect, log_posterior)
+
+# define limits of validity for the MC scattering model
+theta_range_default = {'min_phi':0.1, 'max_phi':0.74, 'min_radius':10., 
+                       'max_radius': 1000., 'min_thickness':1., 'max_thickness':1000., 
+                       'min_l0_r':0, 'max_l0_r':1, 'min_l1_r':-1, 'max_l1_r':1,
+                       'min_l0_t':0, 'max_l0_t':1, 'min_l1_t':-1, 'max_l1_t':1}  # radius in nm, thickness in um
```

Comment:
It is probably worth describing in comments why these values are chosen - why are they hard limits?